package.json

```diff
@@ -61,5 +64,8 @@
     "All": {
       "description": "Test with any doctype."
     }
+  },
+  "dependencies": {
+    "pouchdb": "6.1.1"
```

Comment:
@kosssi could you explain why you had this dependency? I though we let the third-party application to add or not pouchdb in its dependencies.